packages/create-app/templates/default-app/packages/app/src/components/Root/Root.tsx

```diff
@@ -60,7 +60,7 @@ const useSidebarLogoStyles = makeStyles({
 
 const SidebarLogo = () => {
   const classes = useSidebarLogoStyles();
-  const { isOpen } = useContext(SidebarContext);
+  const { isOpen } = useSidebar();
```

Comment:
It sucks but I do feel we need to make this backwards compatible, aka deprecate `SidebarContext`. What do you think @backstage/maintainers?

Reply:
Makes sense to me.  Added in latest changes!